### PR TITLE
Livewrapped bid and analytics adapter

### PR DESCRIPTION
**Pull request by bjorn-lw**
## Type of change

- [X ] New bidder adapter   

## Description of change

- test parameters for validating bids
```
{
  bidder: 'livewrapped',
  params: {
    adUnitId: '6A32352E-BC17-4B94-B2A7-5BF1724417D7'
  }
}
```
- bjorn@livewrapped.com
- [X ] official adapter submission

https://github.com/prebid/prebid.github.io/pull/987

### CODE REVIEW

**Comment by bjorn-lw** [comment]
I'm a bit confused to why all tests fail. Obviously they're passing locally just fine. I'd be very happy for a hint in the right direction so I can fix this as I'm not certain on how to correct this.

**Comment by bretg** [comment]
The main bidder doc at https://github.com/prebid/prebid.github.io/pull/987 is fine, but you may want to have analytics docs as well. See https://github.com/prebid/prebid.github.io/pull/986 for an example.

**Comment by bjorn-lw** [comment]
Running both the bidder and analytics will need setup and separate instructions by us so it didn’t get any docs here, but I think you’re right, I should write something. Will fix, thanks!

File: modules/livewrappedAnalyticsAdapter.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+let initOptions;
+export const BID_WON_TIMEOUT = 500;
+
+const cache = {
+  auctions: {},
+};
+
+let livewrappedAnalyticsAdapter = Object.assign(adapter({EMPTYURL, ANALYTICSTYPE}), {
+  track({eventType, args}) {
+    utils.logInfo('"LIVEWRAPPED_EVENT:', [eventType, args]);
```
[the remark this answers is not in the document]
Comment (bjorn-lw) [reply modules/livewrappedAnalyticsAdapter.js]: Good catch! I fix! (though it's only a debug message :) )

**Comment by bjorn-lw** [comment]
@jsnellbaker  Not sure how to test the analytics adapter on your end, but you could try passing in the publisher id '64C01620-FA98-4936-9794-6001D8EBFDB0'. Thanks!

**Comment by jsnellbaker** [comment]
@bjorn-lw Thanks for the quick reply.  I tried that alternate `publisherId` in the analytics config (see below for reference) on the hello_world page and it worked.

```
pbjs.enableAnalytics({
            provider: 'livewrapped',
            options: {
              publisherId: '64C01620-FA98-4936-9794-6001D8EBFDB0'
            }
          });
```

**Pull request merged**